### DISCUSSION TITLE
Welcome to carbon-components-angular Discussions!

### DISCUSSION THREAD

**Discussion by cal-smith** [IBM/carbon-components-angular#1654] [General]
## 👋 Welcome!
  We’re using Discussions as a place for IBMers and community members to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Solve issues.
  * Learn what the core team is planning
  * And more!

**Comment by sakshi-c** [comment]
how to use fileuploader in a ngx- angular project

**Reply by cal-smith** [reply]
Try opening a new thread with specific details on what you have tried and where you're running into blockers

**Comment by mak009** [comment]
How to validate the reactive form using angular carbon? Not found any example in story or any component related document. please help..!!
